elijah/cards-template1b.js

```diff
@@ -0,0 +1,173 @@
+// Error-detecting version
+
```

Comment:
Looking good. Your checker is doing a lot of the work for validity, but not quite as much as it could. Consider the line that is repeating through your code: __if (!(card%1) && card >= 0 && card < 52)__

Can that be added to your checker function?